autotest/utilities/data/curvepolygon_epsg_32632.csv

```diff
@@ -0,0 +1,3 @@
+_WKT_CURVEPOLYGON_EPSG_32632
```

Comment:
Out of curiosity: The original test data is in EPSG:25832 - do you opted here for EPSG:32632 to have the same geodetic system as EPSG:4326?

Reply:
> Out of curiosity: The original test data is in EPSG:25832 - do you opted here for EPSG:32632 to have the same geodetic system as EPSG:4326?

yes, to make the test independent of potential datum shifts between ETRS89 and WGS 84 which could occur at some point